src/locales/strings/de.json

```diff
@@ -311,6 +311,8 @@
   "title_change_pin": "PIN ändern",
```

Comment:
You can ignore these translated files, they should just get overwritten by the Crowdin translations